src/index.ts

```diff
@@ -181,6 +193,31 @@ export default class Position {
 		return weightedPlacements[0].placement;
 	}
 
+	findBestPlacementAt(
+		offset: Offset,
+		reference: Element,
+		target: Element,
+		placements: string[]) {
+		const positionAt = (_: any, target: Element, placement: string) => {
+			return this.findPositionAt(offset, target, placement);
+		};
+
+		const containerFunction = (): ReferenceRect => ({
```

Comment:
since we don't know what form `reference` could take this should be a argument after `placements` like in `findBestPlacement`